PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -90,6 +90,8 @@
         'keep GenLumiInfoHeader_generator_*_*',
         'keep GenLumiInfoProduct_*_*_*',
         'keep GenEventInfoProduct_generator_*_*',
+        'keep recoGenParticles_genPUProtons_*_*', 
+        'keep recoGenParticles_*_genPUProtons_*',
```

Comment:
why there are two collections?

Reply:
these can/should be changed to be an EDAlias so that the only saved branches are ```recoGenParticles_genPUProtons```

See examples in SimGeneral/MixingModule/python/aliases_PreMix_cfi.py
it will be something like this (perhaps for both ```mix``` and ```mixData```)
```python
genPUProtons = cms.EDAlias(
    mixData = cms.VPSet(cms.PSet(
        type = cms.string('recoGenParticles')
    ))
)
```
I tried this in 250202.17 (premix) and it worked:
- default: ```  recoGenParticles_mixData_genPUProtons_HLT.```
- with the modification (and keep of only ```recoGenParticles_genPUProtons__*```) : ```recoGenParticles_genPUProtons_genPUProtons_HLT```